community-config.yml

```diff
@@ -123,5 +124,6 @@ presets:
     onc_sl_client_id: SAMPLE_CONFIDENTIAL_CLIENT_ID
     onc_sl_confidential_client: true
     onc_sl_client_secret: SAMPLE_CONFIDENTIAL_CLIENT_SECRET
+    patient_ids: "76,184"
```

Comment:
Testing locally it doesn't look like Inferno Community supports this.  This looks like something that was developed in Program, but hasn't been ported to Community.

[TestingInstance in inferno-program has the patient_ids property](https://github.com/onc-healthit/inferno-program/blob/development/lib/app/models/testing_instance.rb)

but,

[TestingInstance in inferno only has patient_id](https://github.com/onc-healthit/inferno/blob/development/lib/app/models/testing_instance.rb)

As a reference, [here's where the presets appear to be set in the code](https://github.com/onc-healthit/inferno/blob/development/lib/app/endpoint/home.rb#L47)

I think we should remove this attribute from the community-preset until it can be supported so that later on we don't assume this is working based on its presence.

Reply:
Good point 👍